src/comms.rs

```diff
@@ -134,6 +137,22 @@ pub enum ApplicationCommand {
         zone_name: StoredName,
         zone_serial: Serial,
     },
+
+    /// Refresh a zone.
+    ///
+    /// The zone loader will initiate a refresh for the zone, and query the
+    /// zone's source to look for a newer version of the zone.  This command
+    /// can be used in response to a user request or a NOTIFY message.
+    RefreshZone {
+        /// The name of the zone to refresh.
+        zone_name: StoredName,
+
+        /// The SOA serial and source address from the NOIFY message.
+        //
+        // NOTE: The serial number is currently ignored.
+        notify_info: Option<(Serial, IpAddr)>,
```

Comment:
Should the IpAddr be optional, in the case of an internally raised event for example?

Reply:
Ah ignore me, the whole notify_info is optional I see.